keystone/common/helper.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2015 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import copy
+from keystone import config
+from keystone import exception
+from keystone.models import token_model
+from keystone.i18n import _LE
+from oslo_log import log
+from oslo_log import loggers
+
+from functools import wraps
+import keystone
+
+LOG = log.getLogger(__name__)
+CONF = config.CONF
+
+projects_to_filter = ('admin', 'service', 'monitoring', 'support')
```

Comment:
All of these are hard-coded projects, roles, and users.  We should not be including data in code...especially data that is highly specific to a singular use case.  This should all be gathered from a configuration file instead.